### PR TITLE
Fix docker cache issue in integrations tests

### PR DESCRIPTION
**Pull request by Sergeykot**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Dockerfile.test [hunk 1]
```diff
@@ -13,15 +13,11 @@ ARG SCC_USERNAME
...
-RUN zypper --non-interactive --gpg-auto-import-keys ref
-RUN zypper --non-interactive up
-RUN zypper --non-interactive in -t pattern devel_osc_build
-RUN zypper --non-interactive install --no-recommend wget curl timezone \
-    gcc-c++ libffi-devel git-core zlib-devel libxml2-devel libxslt-devel libmariadb-devel \
-    mariadb-client mariadb ruby2.5-rubygem-bundler make build sudo ruby-devel nginx
-
+RUN mkdir /tmp/rmt-server
+WORKDIR /tmp/rmt-server
+ADD . /tmp/rmt-server
```
Comment (ikapelyukhin) [inline Dockerfile.test]: Adding the whole directory probably would invalidate the cache if anything changes in it, which would also invalidate the cache for the following steps.

Comment (Sergeykot) [reply Dockerfile.test]: any suggestions?

Comment (ikapelyukhin) [reply Dockerfile.test]: Copy only `integration/install.sh` and not the whole directory.

**Comment by ikapelyukhin** [comment]
1. Is running bundle actually needed? The makefile should run bundler in order to create the tarball.

https://github.com/SUSE/rmt/blob/43ad9b704ccc15ccac5d44859157c7fe08af3fbb/Dockerfile.test#L56

2. I would suggest moving `integration.sh` and maybe `Dockerfile.test` into `integration` directory.
3. Feel free to create scripts for writing `.oscrc` and `rmt.conf` if you like, I guess it would look nicer.

**Comment by Sergeykot** [comment]
running bundle is needed. otherwise it can not run integration tests

**Pull request merged**